src/Graphql/Type/SchemaBuilder.php

```diff
@@ -326,9 +328,13 @@ private function getResourceObjectTypeFields(string $resource, bool $input = fal
     {
         $fields = [];
         $idField = ['type' => GraphQLType::id()];
+        $clientMutationId = GraphQLType::nonNull(GraphQLType::string());
```

Comment:
clientMutationId can be null: https://facebook.github.io/relay/graphql/mutations.htm#sec-Mutation-inputs

Reply:
It can also be not null:

> That input object type may contain an argument named clientMutationId. If provided, that argument must be a String. That argument may be non‐null.

And it is in their official examples: https://facebook.github.io/relay/docs/en/graphql-server-specification.html#mutations

I propose to be strict by default (= to keep this as is).